index.js

```diff
@@ -1 +1,26 @@
-// Your code here
+const myVar = setInterval(function(){ clock() }, 1000);
```

Comment:
You don't need the anonymous function here.  instead you could put in:  `setInterval(clock, 1000);`